### PR TITLE
Laravel 10

### PR DESCRIPTION
**Pull request by mustorze**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "atymic/twitter",
     "description": "Twitter API for PHP & Laravel",
-    "keywords": ["twitter", "laravel", "laravel5", "laravel6"],
+    "keywords": ["twitter", "laravel", "laravel5", "laravel6", "laravel7", "laravel8", "lareavel9", "laravel10"],
```
Comment (atymic) [inline composer.json:4]:
```suggestion
    "keywords": ["twitter", "laravel"],
```

File: composer.json [hunk 2]
```diff
@@ -17,9 +17,9 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^8.1",
```
Comment (atymic) [inline composer.json:20]:
```suggestion
        "php": "^8.0",
```

**Review by atymic — CHANGES_REQUESTED** [review]
See suggestions

File: composer.json [hunk 3]
```diff
@@ -17,9 +17,9 @@
...
         "ext-json": "*",
-        "illuminate/support": "6 - 9",
+        "illuminate/support": "6 - 10",
```
Comment (atymic) [inline composer.json:22]:
```suggestion
        "illuminate/support": "^7.0 || ^8.0 || ^9.0 || ^10.0",
```

**Review by atymic — CHANGES_REQUESTED** [review]

**Comment by mustorze** [comment]
@atymic ✔

**Comment by dantaylorseo** [comment]
When will this be merged please?

**Review by atymic — APPROVED** [review]

**Pull request merged**